Userland/Games/Chess/ChessWidget.cpp

```diff
@@ -535,6 +536,21 @@ void ChessWidget::playback_move(PlaybackDirection direction)
     update();
 }
 
+void ChessWidget::update_move_display_widget(Chess::Board& board)
+{
+    size_t turn = 1;
+    StringBuilder sb;
+    for (auto [i, move] : enumerate(board.moves())) {
+        if ((i + 1) % 2 == 1) {
```

Comment:
nit: This is the same as:
```suggestion
        if (i % 2 == 0) {
```

Reply:
Oh duh, thanks for catching that